packages/server/test/handler.spec.js

```diff
@@ -107,14 +106,12 @@ test('checks service id', async () => {
   const server = Server.create({
     id: w3,
     service: { identity: Access },
-    decoder: CAR,
-    encoder: CBOR,
+    codec: CAR.inbound,
   })
 
   const client = Client.connect({
     id: w3,
-    encoder: CAR,
-    decoder: CBOR,
+    codec: CAR.outbound,
```

Comment:
It would be great if client could interrogate `options.channel` to pick a good default for this if possible. (e.g. if `options.channel` is an HTTP Channel, it could issue an HTTP request to the URL interrogating 'which codecs do you support' e.g. `options.channel.supportedCodecs(clientSupports)` to get back a supported codec intersection)

(we could ship CAR codec with client so that always works, and if someone passes more supported codecs, it works as long as the supported codecs intersection has at least 1)

Reply:
Not saying we should do now, but I hope one day we can make creating ucanto clients not require extra a priori knowledge about how which codecs are required.

Reply:
By reading further I think I realized that the `outbound` object is already set up to do something like that!
cool. I'd be down for `CAR.outbound` to just be a default value here